### PR TITLE
Add some headers to Pune file

### LINKED ISSUE #30: ykeni tracks progress on GitHub Flow
- [x] create a **branch** (git)
- [x] add some **commits** (git)
- [x] open a *pull request* (GitHub)
- [x] additional **commits** (git)
- [ ] _merge_ (git)

### PR DESCRIPTION
**Pull request by ykeni**
The PR adds an Pune file to the class repo. This PR closes #30 

 - @rkparikh do you know any good restaturants in Pune ? 
- If you have been to Goa before ?

### CODE REVIEW

File: ykeni-pune.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Pune, India
+
+### Restaurants
+
+### Attractions for Tourists
+
+- Shaniwarwada
+- Phoenix maill
```
Comment (rkparikh) [inline ykeni-pune.md]: typo error :smile:

**Comment by ykeni** [comment]
@rkparikh I have taken care of your comment

**Comment by ykeni** [comment]
I will merge when I get atleast one thumbs up

**Comment by mrtushardhole** [comment]
:+1: :ship: :shipit:

**Pull request merged**